x-pack/plugins/lens/public/datasources/form_based/operations/definitions/terms/include_exclude_options.tsx

```diff
@@ -99,68 +99,77 @@ export const IncludeExcludeRow = ({
     selectedOptions: IncludeExcludeOptions[],
```

Comment:
This works great already, but I am wondering if we could make one tiny adjustment. When we have some options selected in the opposite option (for example we have Women's Clothing chosen in `include`) maybe we shouldn't display it in exclude? WDYT @MichaelMarcialis?

Technically we would have to filter out the options that are in the opposite bucket so they won't display when we `setTermsOptions` 

<img width="420" alt="Screenshot 2024-11-05 at 10 16 09" src="https://github.com/user-attachments/assets/09b339a6-9aa8-4fb1-934d-0d81f7f78569">